docs/src/main/resources/Managing.adoc

```diff
@@ -1168,6 +1168,280 @@ TLS_RSA_WITH_AES_128_CBC_SHA
 
 |===
 
+=== Configuring {branding} with New Certificates
+{branding} ships with default certificates configured to identify the local machine as "localhost." This allows one to unzip {branding} and run immediately in a secure manner. However, in order to access the {branding} instance from another machine over HTTPS (now the default for many services) without receiving warnings or failures, the default certificates need to be replaced with a certificate reflecting the name of the host machine as reachable from other network nodes.
+
+[IMPORTANT]
+====
+It is important when configuring a new certificate that you follow in order the procedure outlined below. Failure to do so could result in locking everyone out of the {branding} instance.
+====
+
+==== Important Terms
+
+[cols="3" options="header"]
+|===
+
+|Term
+|Definition
+|Example
+
+|FQDN	
+|Fully Qualified Domain Name	
+|search.codice.org
+
+|{branding}_HOME	
+|The path to the unzipped {branding} distribution	
+|/opt/ddf/ddf-2.6.0
+
+|Alias	
+|The nickname given to a certificate within a keystore to make it easily identifiable	
+|localhost within the default {branding} serverKeystore.jks
+
+|CN	
+|Common Name - The FQDN of the server as defined within the Certificate.	
+|search.codice.org
+
+|===
+
+==== Changing the URLs
+===== Updates via Web Configuration
+It is best to first change the URLs in {branding} before changing the certs. This allows {branding} to run in it's default state while everything gets updated. Once everything is updated, simply stop {branding}, update the keystores, and start it back up.
+
+. Start DDF 
++
+[source,bash]
+----
+$> cd <DDF_HOME>/bin
+$> ./ddf
+----
++
+. Go to https://localhost:8993/admin and step through the installer leaving all the defaults.
```

Comment:
They should update the hostname and port with the actual hostname and desired port for the system they are deploying.